batchflow/utils.py

```diff
@@ -150,8 +150,12 @@ def show_research(df, layout=None, average_repetitions=False, log_scale=False,
         Size of rolling window.
     color: sequence of matplotlib.colors, optional
         Colors for plots would be randomly sampled from given set.
-    scale: tuple, default: (9, 7)
-        Scaling factors for the figure.
+    kwargs:
```

Comment:
That is prone to errors: if you pass anything that `subplots` can't use, it will raise an error. Better to use separate dicts for that

Reply:
There are already a lot of parameters in the function. And it just shows subplots. There's no need to wrap kwargs for plt.subplots.
I suppose that no one doesn't want to make a function with tens of parameters with `show_research` inside.